inc/Engine/WPRocketUninstall.php

```diff
@@ -217,6 +233,12 @@ private function delete_plugin_data() {
 		array_walk( $this->transients, 'delete_transient' );
 		array_walk( $this->options, 'delete_option' );
 
+		foreach ( $this->dynamic_transients as $transient ) {
+			$wpdb->query(
+				$wpdb->prepare( "DELETE FROM $wpdb->options WHERE option_name LIKE %s", '%' . $transient . '%' )
```

Comment:
Um, I made a loop and a variable because I was thinking it would be easier to maintain in the future if we get more dynamic transient. We would only need to add an entry within the array https://github.com/wp-media/wp-rocket/blob/3b10ea8efac887946dca89b0118a19b9aa4ab742/inc/Engine/WPRocketUninstall.php#L87

 @Tabrisrp what is your thought about this ?

Reply:
I've pushed a commit replacing the `foreach` loop by a `REGEXP` :)